crates/chia-puzzles/src/puzzles/cat.rs

```diff
@@ -250,77 +223,4 @@ mod tests {
         assert_puzzle_hash!(EVERYTHING_WITH_SIGNATURE_TAIL_PUZZLE => EVERYTHING_WITH_SIGNATURE_TAIL_PUZZLE_HASH);
         assert_puzzle_hash!(GENESIS_BY_COIN_ID_TAIL_PUZZLE => GENESIS_BY_COIN_ID_TAIL_PUZZLE_HASH);
     }
-
-    #[test]
-    fn curry_cat_tree_hash() {
```

Comment:
don't we still need these tests?

Reply:
The special cased functions for currying these puzzles are gone, so I removed the tests for them as well. We could technically test the use of `CurriedProgram` but it seemed redundant to me since it's the same as using the `Allocator` now, just with a different encoder.

And you could theoretically use it with all of the puzzles now. Open to other ideas for testing though